api/v1/flow/flow.proto

```diff
@@ -183,6 +185,18 @@ enum TraceObservationPoint {
     TO_NETWORK = 11;
 }
 
+enum TraceReason {
```

Comment:
Is there a reason behind the ordering here? Intuitively I'd expect `TRACE_REASON_UNKNOWN` to be `0` (or maybe `TRACE_REASON_UNSPECIFIED` to follow [proto best practices](https://protobuf.dev/programming-guides/dos-donts/#unspecified-enum)).